src/applications/vaos/containers/ReviewPage.jsx

```diff
@@ -0,0 +1,58 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import { connect } from 'react-redux';
+import {
+  getFormData,
+  getChosenFacilityInfo,
+  getChosenClinicInfo,
+} from '../utils/selectors';
+import ReviewDirectScheduleInfo from '../components/ReviewDirectScheduleInfo';
+import ReviewRequestInfo from '../components/ReviewRequestInfo';
+import LoadingButton from 'platform/site-wide/loading-button/LoadingButton';
+
+export class ReviewPage extends React.Component {
+  render() {
+    const { data, facility, clinic } = this.props;
+
+    return (
+      <div>
+        {data.isDirectSchedule && (
```

Comment:
I'm assuming we'll set something like this if we hit all the criteria for direct scheduling.